Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG buildImage=node:16.13.0-alpine
+
+# BE
+FROM $buildImage as be-build-stage
+WORKDIR /usr/src/app/
+COPY ./be/package*.json ./
+RUN npm ci
+COPY ./be .
+COPY ./openapi.yaml ../
+ENV NODE_ENV=production
+RUN apk add openjdk11
+RUN npm run generate:build
+RUN npm prune
+
+# FE
+FROM $buildImage as fe-build-stage
+WORKDIR /usr/src/app/
+COPY ./fe/package*.json ./
+RUN npm ci
+COPY ./fe/ .
+COPY ./openapi.yaml ../
+ENV NODE_ENV=production
+RUN apk add openjdk11
+RUN npm run generate:build
+RUN npm prune
```

Comment:
Given that the two build-stages look _very_ similar, I guess it makes sense to merge them into one.